packages/yoastseo/src/scoring/assessments/inclusiveLanguage/InclusiveLanguageAssessment.js

```diff
@@ -36,7 +36,7 @@ export default class InclusiveLanguageAssessment {
 	 * @returns {void}
```

Comment:
Also good to add the description of these parameters:  

	 * @param {string} [config.ruleDescription] A description of the rule.
	 * @param {string} [config.category] The category of the assessment.